FSJavascript/Sign-UpForm/styles.css

```diff
@@ -0,0 +1,191 @@
+:root {
+  --white: rgb(238, 236, 236);
+  --light-gray: #f1f1f1;
+  --med-gray: #dcdddd;
+  --dark-gray: #3b3b3b;
+  --primary: #596d48;
+  --primary-emphasis: #283121;
+  --blue: blue;
+  --red: red;
+  font-family: sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+}
+
+a:link,
+a:visited {
+  color: unset;
+}
+
+em {
+  font-style: italic;
+}
+
+h2 {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.title {
+  flex: 1;
+  height: 100vh;
+  background: url(./assets/halie-west-25xggax4bSA-unsplash.jpg) 0% 0% / cover;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100px;
+  position: relative;
+  top: 200px;
+  padding: 16px;
+  background-color: #00000052;
```

Comment:
What is this, alpha channel in hex code? I've never seen that done usually I just see the rgba() function